CMakeSettings.json

```diff
@@ -0,0 +1,40 @@
+﻿{
+  "configurations": [

```

Comment:
I did not understand. Which file? CMakeSettings.json?

Reply:
I mean we have two spaces in the beginning of the line before `"configurations"`, instead of tabs/four spaces/whatever we have in `tasks.vs.json`. I think we should keep one formatting style at least among JSON files.

Reply:
Actually, we have the same formatting in `tasks.vs.json`

Reply:
It seems github shows `tasks.vs.json` with incorrect indentations. If open this file in VS Code or Visual Studio the indentations are correct